docs/quick-reference.adoc

```diff
@@ -1 +1,66 @@
 = Sylt Quick Reference
+
+Here you will find most things in Sylt in a concise format.
+
+.Variables
+[source, sylt]
+----
+constant :: 3.14
```

Comment:
Maybe `pi_constant`? Or something?

Reply:
I would say no. I could change the value if it is distracting.